src/vm/gdbjit.cpp

```diff
@@ -2547,21 +2597,25 @@ void NotifyGdb::OnMethodPrepared(MethodDesc* methodDescPtr)
     jit_symbols->symfile_addr = symfile_addr;
     jit_symbols->symfile_size = symfile_size;
 
-    /* Link into list */
-    jit_code_entry *head = __jit_debug_descriptor.first_entry;
-    __jit_debug_descriptor.first_entry = jit_symbols;
-    if (head != 0)
     {
-        jit_symbols->next_entry = head;
-        head->prev_entry = jit_symbols;
-    }
+        CrstHolder _crst(&JitDescriptorCrst);
```

Comment:
Naming convention - could you please remove the leading underscore from the name?

Reply:
done